src/evaluation/config/retriever_eval_squad_config.py

```diff
@@ -1,10 +1,11 @@
 parameters = {
-    "k": [3],
-    "retriever_type": ['bm25'],
+    "k": [1, 3],
+    "retriever_type": ['epitca', 'bm25'],
     "google_retriever_website": ['service-public.fr'],
     "squad_dataset": [
         "./clients/cnil/knowledge_base/squad.json"
     ],  # data/evaluation-datasets/fquad_valid_with_impossible_fraction.json data/evaluation-datasets/testing_squad_format.json
+    "epitca_perf_file": ["./clients/cnil/knowledge_base/raw_and_preparation/epitca_perf_V2.json"],
```

Comment:
Maybe here write a comment to explain what to do when you don't want to use eptica perf file

Reply:
Done.